src/modules/75-cf/components/FlagActivation/LinkJiraIssueModal.tsx

```diff
@@ -0,0 +1,190 @@
+import patch from '../../utils/instructions'
+
+export interface LinkJiraIssueModalProps {
+  featureIdentifier: string
+  refetchFlag: () => void
+}
+
+const LinkJiraIssueModal: FC<LinkJiraIssueModalProps> = ({ featureIdentifier, refetchFlag }) => {
+  const { showError, showSuccess } = useToaster()
+
+  const { accountId: accountIdentifier, orgIdentifier, projectIdentifier } = useParams<Record<string, string>>()
+  const { getString } = useStrings()
+  const { activeEnvironment } = useActiveEnvironment()
+
+  const [isModalDisplayed, setIsModelDisplayed] = useState<boolean>(false)
+  const hideModal = useCallback(() => setIsModelDisplayed(false), [])
+  const openModal = useCallback(() => setIsModelDisplayed(true), [])
+
+  const queryParams = {
+    projectIdentifier,
+    environmentIdentifier: activeEnvironment,
+    accountIdentifier,
+    orgIdentifier
+  }
+
+  const { mutate, loading: patchLoading } = usePatchFeature({
+    identifier: featureIdentifier,
+    queryParams
+  })
+
+  const {
+    data,
+    loading: jiraIssuesLoading,
+    refetch
+  } = useGetJiraIssues({
+    lazy: true,
+    debounce: 700
+  })
+
+  const jiraItems = useMemo<SelectOption[]>(() => {
+    if (data?.issues?.length) {
+      return data.issues.map(issue => ({ label: issue.key as string, value: issue.key as string }))
+    }
+    return []
+  }, [data?.issues])
+
+  const handleSubmit = (jiraIssueKey: string): void => {
+    patch.feature.addInstruction({
+      kind: 'addJiraIssueToFlag',
+      parameters: {
+        issueKey: jiraIssueKey as string
+      }
+    })
+
+    patch.feature.onPatchAvailable(async instructions => {
+      try {
+        await mutate(instructions)
+        hideModal()
+        refetchFlag()
+        showSuccess(getString('cf.featureFlags.jira.successMessage'))
+      } catch (e) {
+        showError(getErrorMessage(e), undefined, 'cf.featureFlags.jira.errorMessage')
+      } finally {
+        patch.feature.reset()
+      }
+    })
+  }
+
+  return (
+    <>
+      <Layout.Horizontal flex={{ justifyContent: 'flex-start' }}>
+        <RbacButton
+          data-testid="add-jira-issue-modal"
+          minimal
+          icon="small-plus"
+          size={ButtonSize.SMALL}
+          font={{ size: 'small' }}
+          style={{ padding: 0 }}
+          text={getString('cf.featureFlags.jira.newJiraIssueButton')}
+          onClick={() => openModal()}
+          permission={{
+            resource: { resourceType: ResourceType.FEATUREFLAG },
+            permission: PermissionIdentifier.EDIT_FF_FEATUREFLAG
+          }}
+          featuresProps={{
+            featuresRequest: {
+              featureNames: [FeatureIdentifier.MAUS]
+            }
+          }}
+        />
+      </Layout.Horizontal>
+
+      <Formik
+        initialValues={{
+          selectedJiraIssueKey: ''
+        }}
+        formName="addJiraIssueDialog"
+        enableReinitialize={true}
```

Comment:
You can just pass this and it'll evaluate as `true`
```suggestion
        enableReinitialize
```